### ISSUE TITLE
Hibernate?!

### ISSUE THREAD

**Issue by lockTP** [infsci2711/MultiDBs-FilesAPIs2DBs-Server#2]
Do we need Hibernate to talk to the database?

**Comment by EvgenyKarataev** [comment]
No, I don't think you need to use hibernate, because you don't have one schema, depending on the input file you target database will be different.
